### PR TITLE
fixed phpstan error until level 5

### PR DESCRIPTION
**Pull request by robinlehrmann**
belongs to #26

### CODE REVIEW

File: src/ClientApi/Actions/SendTransfer/Action.php [hunk 1]
```diff
@@ -532,9 +532,7 @@ protected function prepareTransfers()
             }
 
             // remove the checksum
-            $transfer->setRecipientAddress(
```
Comment (Techworker) [inline src/ClientApi/Actions/SendTransfer/Action.php:535]: `removeChecksum` returns a new instance of the address without a checksum.

File: src/RemoteApi/Commands/GetBalances/Request.php [hunk 2]
```diff
@@ -72,7 +72,7 @@ public function setAddresses(array $addresses): self
      */
     public function addAddress(Address $address): self
     {
-        $this->addresses[] = $address->removeChecksum();
```
Comment (Techworker) [inline src/RemoteApi/Commands/GetBalances/Request.php]: `removeChecksum` returns a new instance of the address without a checksum.

**Comment by Techworker** [comment]
Thanks for that, see my comments related to the `removeChecksum`method and the changes you introduced

File: src/ClientApi/Actions/SendTransfer/Action.php (hunk 1 above)
Comment (robinlehrmann) [reply src/ClientApi/Actions/SendTransfer/Action.php:535]: Okay, you are right. I've fixed it, but I think we should refactor it and don't remove checksum at this point.

File: src/RemoteApi/Commands/GetBalances/Request.php (hunk 2 above)
Comment (robinlehrmann) [reply src/RemoteApi/Commands/GetBalances/Request.php]: See other comment, I prefer to refactor it and put it at an other point of the code.

File: src/RemoteApi/Actions/GetBalances/Action.php [hunk 3]
```diff
@@ -72,7 +75,7 @@ public function setAddresses(array $addresses): self
      */
     public function addAddress(Address $address): self
     {
-        $this->addresses[] = $address->removeChecksum();
```
Comment (Techworker) [inline src/RemoteApi/Actions/GetBalances/Action.php:75]: Thats not correct, if someone calls addAddress with an address including the checksum the checksum is not removed.

**Review by Techworker — CHANGES_REQUESTED** [review]

File: src/RemoteApi/Actions/GetBalances/Action.php (hunk 3 above)
Comment (robinlehrmann) [reply src/RemoteApi/Actions/GetBalances/Action.php:75]: fixed

Comment (Techworker) [reply src/RemoteApi/Actions/GetBalances/Action.php:75]: Now it's removing it twice when you call `setAddresses`

**Review by Techworker — APPROVED** [review]
Looks fine to me, thanks.

**Pull request merged**